MediaStorageSample/README.md

```diff
@@ -1,15 +1,18 @@
 # Media Storage Sample
 * In this sample, you can list your Media IDs.
 
-## Setup
-* Complete [Setup](https://github.com/ricohapi/media-storage-swift#setup) of Media Storage for Swift.
+## Using the Sample
+* Clone Ricoh Media Storage for Swift by running the following commands:
+```sh
+$ git clone --recursive https://github.com/ricohapi/MediaStorage.git
+```
+* Open `MediaStorage.xcodeproj` in the `MediaStorage` directory, and open `MediaStorage.xcodeproj`.
+* Choose RicohAPIAuth scheme at the scheme menu of Xcode and run it.
+* Choose MediaStorage scheme at the scheme menu of Xcode and run it.
```

Comment:
This procedure has found to be unnecessary. : [NECO-90](https://pflabo.atlassian.net/browse/NECO-90)